beep/structure/base.py

```diff
@@ -200,6 +200,8 @@ def __init__(self, raw_data, metadata, paths=None, schema=None, impute_missing=T
 
             if os.path.exists(abs_schema):
                 self.schema = abs_schema
+            elif os.path.exists(os.path.join(VALIDATION_SCHEMA_DIR, os.path.split(schema)[-1])):
+                self.schema = os.path.join(VALIDATION_SCHEMA_DIR, os.path.split(schema)[-1])
```

Comment:
may cause silent conflicts

Reply:
Could you elaborate a little on the conflicts that you anticipate?

Reply:
In env1, `/path1/to/some_schema.yaml` and the file contains schema A
In env2, `/path2/to/some_schema.yaml` and the file contains schema B, just coincidentally under the same name

Perhaps not of grave concern, but just looking ahead here

Also I might have misunderstood how this is working